examples/Extension/AcmeExtension.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace PhpBench\Examples\Extension;
+
+use PhpBench\DependencyInjection\Container;
+use PhpBench\DependencyInjection\ExtensionInterface;
+use PhpBench\Examples\Extension\Command\CatsCommand;
+use PhpBench\Examples\Extension\Executor\AcmeExecutor;
+use PhpBench\Examples\Extension\ProgressLogger\CatLogger;
+use PhpBench\Examples\Extension\Report\AcmeGenerator;
+use PhpBench\Extension\CoreExtension;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+// section: command_di,executor_di,progress_logger_di,report_generator_di
```

Comment:
instead of this can just pass mutiple sections, one of which can be `common` f.e.